src/common/links.ts

```diff
@@ -51,3 +52,31 @@ export function isValidHttpUrl(link: string): boolean {
     return false;
   }
 }
+
+export const getShortUrl = async (url: string | URL): Promise<string> => {
```

Comment:
Yeah, I actually prefer URL constructor (as seen in this PR), but let's make it `string` for now since I think it looks clunky to accept 2 types and return single type.